Jupyter notebook example

komalali opened the pull request.

EvanBoyle reviewed (approved):
I just ran through this and __wow__ is this seriously cool or what? 

We definitely need to write a blog post and tweet about this. 

My only suggestion might be to include a screenshot to include in the readme (I'll attach one below). Might also be a good idea to include some step by step instructions (run each cell in order). Took me a second to figure out this was what I needed to do having been a while since I used jupyter. 
![Screen Shot 2021-02-09 at 3 33 34 PM](https://user-images.githubusercontent.com/2553171/107442778-79030800-6aec-11eb-9083-3d5d874d8993.png)

komalali commented:
@EvanBoyle I just added a db migration example and fixed up the README. Would you mind running through the db example on your machine just to make sure everything runs properly and then I can merge this?

EvanBoyle reviewed (approved):
I ran through the migration example. Works like a charm!

I would suggest one small addendum to the README now that there are two notebooks in this example. Mainly, upfront put a description of the two notebooks along with how long it will take.

```markdown
This example contains two notebooks:

1. `automation_api.ipynb`: (est. runtime 3 minutes) A simple overview of Automation API that demonstrates doing a basic deployment of a static website from within a Jupyter notebook. 
2. `database_migration.ipynb`: (est. runtime 15 minutes) Set up data science infrastructure from your notebook. We create an AWS RDS SQL database, configure the schema, and load data.
```
